extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -403,7 +403,9 @@ describe('amp-a4a', () => {
         const doc = fixture.doc;
         const a4aElement = doc.createElement('amp-a4a');
         const a4a = new AmpA4A(a4aElement);
-        return a4a.maybeRenderAmpAd_(null).then(rendered => {
+        a4a.adUrl_ = 'http://foo.com';
+        a4a.maybeRenderAmpAd_ = function() { return Promise.resolve(false); }
```

Comment:
@tdrl I agree that the output from validateAdResponse_ can be null but I think the creative && ... check is sufficient for now.  As we have discussed offline, we need to rework the flow to handle empty creative (either from empty XHR response or empty creative from extract).  I believe our plan was to check the state at each point within the promise chain flow as opposed to within each private function call (in other words as we are doing now as opposed to allowing empty creative state to be passed to validateAdResponse_).